### PR TITLE
Include text field and prevent crash

### PR DESCRIPTION
**Pull request by jmcarcedo**
### Checklist

- [x] I've validated if any R8 rule should be added to the relevant proguard-rules.pro file. More info about it [on Confluence](https://futureworkshops.atlassian.net/wiki/spaces/FMW/pages/2319187983/Validate+R8+rules+for+core+and+plugins)
- [x] If I'm updating a plugin submodule, I'm sure that the reference on the submodule is on `main`

### Task

[Charts / Dashboard step](https://3.basecamp.com/5245563/buckets/26145695/todos/4870042132)

### Feature/Issue

Include text field for every chart type and prevent crash

### Implementation

* Add text field to base dashboard item and assign it the text field recovered from the json
* Avoid crash by only managing `chartValues` as possible values for line, pie and bar charts

### CODE REVIEW

File: charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/ItemProvider.kt [hunk 1]
```diff
@@ -87,9 +87,10 @@ sealed class ItemProvider {
                             listItemId = it.id,
                             chartType = it.chartType,
                             title = localizationService.getTranslation(it.title),
+                            text = it.text,
```
Comment (igorcferreira) [inline charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/ItemProvider.kt]: @jmcarcedo Shouldn't this text also be translated?

Comment (jmcarcedo) [reply charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/ItemProvider.kt]: @igorcferreira you are correct, changed in [fca2162](https://github.com/FutureWorkshops/MWChartsPlugin-Android/pull/13/commits/fca21627e9541c4774df647a3f95c467585e30d1)

Comment (igorcferreira) [reply charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/ItemProvider.kt]: @jmcarcedo I saw that the commit linked here fixes the text for the `onItemsReady` method. But, it does not apply the same for this `load` method.

Comment (jmcarcedo) [reply charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/ItemProvider.kt]: Yeah sorry for that I got distracted. Changed in [e71a216](https://github.com/FutureWorkshops/MWChartsPlugin-Android/pull/13/commits/e71a2169b8755bd0858f59bd226a282ae674d8eb)

**Review by igorcferreira — APPROVED** [review]

**Pull request merged**